core/src/test/java/org/elasticsearch/index/query/RandomQueryBuilder.java

```diff
@@ -51,6 +54,21 @@ public static QueryBuilder createQuery(Random r) {
     }
 
     /**
+     * Create a new multi term query of a random type
+     * @param r random seed
+     * @return a random {@link MultiTermQueryBuilder}
+     */
+    public static MultiTermQueryBuilder createMultiTermQuery(Random r) {
+        // for now, only use String Rangequeries for MultiTerm test, numeric and date makes little sense
+        // see issue #12123 for discussion
+        // Prefix / Fuzzy / RegEx / Wildcard can go here later once refactored and they have random query generators
+        RangeQueryBuilder query = new RangeQueryBuilder(BaseQueryTestCase.STRING_FIELD_NAME);
+        query.from("a"+RandomStrings.randomAsciiOfLengthBetween(r, 1, 10));
+        query.to("z"+RandomStrings.randomAsciiOfLengthBetween(r, 1, 10));
+        return query;
```

Comment:
I tried this but having the RangeQueryBuilderTest setup calling the utility class just for one of the three cases covered there looks odd. Too much indirection just to save three lines, if I'm honest. Would like to leave it like this for now, so only non-leaf queries need to use RandomQueryBuilder for nested queries, and try to extend this method with other refactored multi term queries later.